### ISSUE TITLE
Plugin does not honor nonProxyHosts System Property

### ISSUE THREAD

**Issue by mikesmithson** [noamt/rest-gradle-plugin#23]
I have configured the nonProxyHosts  system property in my ~/.gradle/gradle.properties file that bypasses the proxy configuration for localhost, etc.  However, when trying interact with a localhost URL using your plugin, I am getting a ' javax.net.ssl.SSLPeerUnverifiedException: peer not authenticated' exception. I can unset the proxy in my gradle build script or comment out the property entries in my gradle.properties file but I don't think I should have to do this. Is there a workaround for this, did I miss something or is this a new feature request?

**Comment by noamt** [comment]
You're right that the plugin should implicitly apply `nonProxyHosts` in this case

**Comment by mikesmithson** [comment]
OK, I can make a pull request.
